ldelf/sys.c

```diff
@@ -72,6 +72,16 @@ TEE_Result sys_copy_from_ta_bin(void *dst, size_t num_bytes, uint32_t handle,
 	return _ldelf_cp_from_bin(dst, offs, num_bytes, handle);
 }
 
+TEE_Result sys_map_zi_and_copy_from_ta_bin(vaddr_t *va, size_t memsz,
```

Comment:
A bit a inline description would be welcome. Maybe in the header file.

Reply:
left inline comments